ci/test_suites/authenticators_gcp/get_tokens_to_files.sh

```diff
@@ -59,9 +59,9 @@ _get_token_to_files() {
     local token=""
 
     if [ -z "$auth_token" ]; then
-      token="$(curl -s -H 'Metadata-Flavor: Google' $token_url)" || exit 1
+      token="$(curl -L -s -H 'Metadata-Flavor: Google' $token_url)" || exit 1
```

Comment:
@royico can you change this to `--location` instead of `-L`? it is best to have the long flags so it's more readable.
i would also change `-s` to `--silent` while you're at it.